include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -226,6 +226,15 @@ typedef enum ExecutionMode {
   ORT_PARALLEL = 1,
 } ExecutionMode;
 
+typedef enum OrtProgrammingProjection {
+  ORT_PROJECTION_CROSSPLAT_C = 0, // default
+  ORT_PROJECTION_CPLUSPLUS,
+  ORT_PROJECTION_CROSSPLAT_CSHARP,
+  ORT_PROJECTION_CROSSPLAT_PYTHON,
+  ORT_PROJECTION_CROSSPLAT_JAVA,
+  ORT_PROJECTION_WINML,
```

Comment:
should we remove the CROSSPLAT prefix?   CPLUSPLUS doesn't have it, we could just call these C, CPLUSPLUS, CSHARP, WINML ....